### ISSUE TITLE
Unstated-next with Next.js and SSR

### ISSUE THREAD

**Issue by Jekins** [jamiebuilds/unstated-next#69]
Hi. How is it possible to use unstated-next along with Next.js? Need SSR for SEO.

**Comment by leonardojimenez** [comment]
Yes, we are using it in a project and both "play" well together :), AFAIK this Unstated-next internally uses React context and hooks, thats it.

**Comment by stfenjobs** [comment]
@Jekins   @leonardojimenez 

https://github.com/zeit/next.js/tree/canary/examples/with-unstated
